README.md

```diff
@@ -1,41 +1,78 @@
 # xk6-disruptor
 
-xk6-disruptor is a [k6](https://k6.io) extension for injecting faults into the system under tests.
+xk6-disruptor is a [k6](https://k6.io) extension for injecting faults into the system under tests. It offers an API for creating disruptors that target one specific type of component (for example,  Pods) and are capable of injecting different types of [faults](#faults) such as errors in HTTP requests served by that component. Currently disruptors exist for [Pods](#pod-disruptor] and [Services](#service-disruptor), but others will be introduced in the future as well as additional types of faults for the existing disruptors.
```

Comment:
> At this time, the project aims to aid developers in building reliable systems in k8s. It's not intended for ops or infrastructure engineers. 

I don't think we should state this because even when it is not the current scope, it is relatively easy to add disruptions that may be of interests of these audiences. In fact, the node disruptor will be likely implemented in the next cycle, as an example.

Reply:
OK. I agree with this pushback. Let's not make claims about the ops/infra people not being the target audience.